### DISCUSSION TITLE
Is svg-to-jsx a bad decision?

### DISCUSSION THREAD

**Discussion by codeart1st** [parcel-bundler/parcel#5853] [Q&A]
Hey I tried out parcel v2 and the included transformer-svg-react package but got disappointed. It messed up some of my SVGs because of missing attributes and wrong ordering of the svg child elements like path and rect. I looked into the svg-to-jsx repository and it's not so popular at all. Should we use a more popular dependency with more supported attributes and a more stable codebase?

https://github.com/janjakubnanista/svg-to-jsx

**Comment by mischnic** [comment]
If the current package has problems, I don't see why not.

Maybe https://www.npmjs.com/package/@svgr/core?

**Comment by codeart1st** [comment]
That would be my favorite too because of two things. First svgr has his own package for parcel v1 https://www.npmjs.com/package/@svgr/parcel-plugin-svgr which is actually the most common way. And second svgr is in my opionen the most popular svg react rendering library.

Is a PR needed and welcome?
